### DISCUSSION TITLE
The system does not boot RPI3B+

### DISCUSSION THREAD

**Discussion by Ssanchez29** [opencardev/crankshaft#646] [Q&A]
Hello, I try to start the system and it gives me an error. I have passed the debug and this is the result it gives me but I cannot identify exactly what is wrong.
[debug.zip](https://github.com/opencardev/crankshaft/files/13198659/debug.zip)

**Comment by wjcarpenter** [comment]
Your kern.log has some instances of "Under-voltage detected!". That can cause all kinds of erratic behavior on RPi. The situation can be caused by either a power supply that is not strong enough or a USB cable that is not able to carry the necessary current. Even with those thing that were fine for your 3B+ before, if they are now also powering a screen they may not be strong enough.
